app/app.go

```diff
@@ -112,10 +108,6 @@ var (
 		upgrade.AppModuleBasic{},
 		evidence.AppModuleBasic{},
 		transfer.AppModuleBasic{},
-		funding.AppModuleBasic{},
-		curating.AppModuleBasic{},
```

Comment:
Isn't this our module?

Reply:
Yeah I removed the funding module as well since we're not going with the bonding curve approach anymore. We can refer to the IBC stuff in git history if we need it for later. We aren't locking ATOM on the chain anymore, and I believe we don't need a separate module to setup IBC channels to do transfers. Of course this is dependent on how we implement the wallet.

Reply:
I meant the curating module removing it from the module manager will break it

Reply:
Oh ooops!